### PR TITLE
moved material from retro-video-store repo

### PR DESCRIPTION
**Pull request by beccaelenzil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+
+Once upon a time in ye olden days to watch a movie a person had to head down to their local video store and rent a video.  We are going to step into the shoes of this old timey retro video store owner and build for ourselves all of the tools that we need in order to run our successful corner video store. 
+
+You have already built a [Video Store API](https://github.com/AdaGold/retro-video-store) for the backend in Part 1 of the project.
+
+In keeping with our retro vibe, for this part of the project we will build a Command Line Interface (CLI) for the front end.
+
+## Manual Tests
+
+We will be testing your CLI manually to see how it performs all of the required behaviors.  We recommend that you find a partner to "play-test" your CLI and put it through it's paces.
```
Comment (jmaddox19) [inline README.md:11]: I'm a little worried about how far we're going in the direction of ambiguity here. Namely, how that might create unhelpful anxiety for students and how that might cause Instructors to get a lot of repeated questions about what is and isn't expected.
I think if we want it to be this open-ended, I might prefer we just say something like "We're not going to review this very strictly, have fun with it! Use your creativity!"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+
+In keeping with our retro vibe, for this part of the project we will build a Command Line Interface (CLI) for the front end.
+
+## Manual Tests
+
+We will be testing your CLI manually to see how it performs all of the required behaviors.  We recommend that you find a partner to "play-test" your CLI and put it through it's paces. 
+
+## Timeline
+
+This project's submission deadline is Friday May 28th, 9:00PM.
```
Comment (jmaddox19) [inline README.md:15]: As I mentioned in the other review, I'm in favor of taking this out until/unless we agree collectively that we want to start having deadlines in READMEs again.

**Review by jmaddox19 — COMMENTED** [review]
Thank you for pulling this out!
This is absolutely just a sounding board for my thoughts on the project and has nothing to do with you splitting this out so feel free to ignore if you want to make this change happen quickly. I can bring my concerns into a different context.

File: README.md (hunk 2 above)
Comment (jbieniosek) [reply README.md:15]: I did not realize we didn't usually including dates in READMEs, but now that you mention it that makes sense, I think taking it out is a good plan :-).

**Review by jbieniosek — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (CheezItMan) [reply README.md:15]: To be fair it's a new thing this cohort.

File: README.md (hunk 1 above)
Comment (CheezItMan) [reply README.md:11]: Yeah, either tell them to go wild, or give them some user stories to complete.

**Review by CheezItMan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jmaddox19) [reply README.md:11]: I mean, this does have clear user stories to complete. My concern is just that there's no further explanation about how to handle edge cases within user stories and what edge cases to worry about

Comment (beccaelenzil) [inline README.md:11]:
```suggestion
## Review

There are no tests for the CLI. Have fun with it! You use use the [user stories](ada-project-docs/part-2/user_stories.md) to guide its functionality. Instructors will review the CLI and provide feedback on how it addresses the user stories and user experience. We recommend that you find a partner to "play-test" your CLI and put it through its paces. 
```

File: README.md (hunk 2 above)
Comment (beccaelenzil) [reply README.md:15]:
```suggestion
```
